src/adr/0001-record-architecture-decisions.md

```diff
@@ -0,0 +1,19 @@
+# 1. Record architecture decisions
+
+Date: 2019-10-02
+
+## Status
+
+Accepted
```

Comment:
Well, I'm concerned... 

This decision have been accepted in a Linagora environment, and not discussed in an Apache environment.

While this 'decision' makes sense in a Linagora environment, I find it inapropriate to expose it on an Apache repository.

We have to clearly debate the meaning of 'accepted' here.

This decision needs at the very list a mailing list proposal.